anago

```diff
@@ -216,11 +216,9 @@ check_prerequisites () {
   # TODO: Users outside google? Ask/derive domain?
   # TODO: The real test here is to verify that whatever auth has access to
   #       do releasey things
-  gcloud_auth_list=$($GCLOUD auth list 2>/dev/null)
   for user in $G_AUTH_USER $userat; do
     logecho -n "Checking cloud account/auth $user: "
-    if [[ "$gcloud_auth_list" =~ -\ $user ]] && \
-       (logrun gcloud config set account $user && \
+    if (logrun gcloud config set account $user && \
```

Comment:
It looks like `gcloud config set account bogus@nowhere.com` returns 0.
Maybe just capture the `gcloud_auth_list=$($GCLOUD auth list --format="value(account))` and look for $user in that list.